File: Programacion/Tema 4/Ejercicios/Ejercicios 1/Ejercicios_01/src/Ejercicio_4.java
import java.util.Scanner;

public class Ejercicio_4 {
    static void tablamulti(int numero){
        for (int i=0;i<=10;i++){
            System.out.println(numero+"*"+i+"="+numero*i);
        }
    }
    public static void main(String[] args) {
        Scanner scanner=new Scanner(System.in);
        System.out.println("Dime el numero ");
        int numero= scanner.nextInt();
        tablamulti(numero);
    }
}

File: Programacion/Tema 7/Ejercicios/Repaso Examen/Repaso_Examen/src/main/kotlin/Ejercicio_4.kt
import java.math.BigInteger
import kotlin.random.Random

fun main() {
    /*
    println("Dime un numero")
    val numero= readLine()?.toInt()
    esprimo(numero)
    */
    val numero= 9
    println(numero.toBigInteger().isProbablePrime(Random.nextInt(1,101)))


}
fun esprimo(numero: Int?){
    /*
    var numerotemporal= numero
    var primo=false;
    if (numero==1){
        println("El numero 1 no es primo")
    }else{
        if (numerotemporal != null) {
            while (numerotemporal>1){
                if (numero != null) {
                    if ((numero/numerotemporal)>0){
                        primo=false

                    }else{
                        primo=true
                    }
                }
                numerotemporal--
            }

        }
    }
    println("El numnero es primo "+primo)
    */
}